cmd/assetsvc/handler.go

```diff
@@ -242,11 +277,19 @@ func getChartVersionSchema(w http.ResponseWriter, req *http.Request, params Para
 
 // listChartsWithFilters returns the list of repos that contains the given chart and the latest version found
 func listChartsWithFilters(w http.ResponseWriter, req *http.Request, params Params) {
-	charts, err := manager.getChartsWithFilters(params["namespace"], params["chartName"], req.FormValue("version"), req.FormValue("appversion"))
+	// Workaround for retrieving chart name from both "chartName" or "name" params
```

Comment:
It was needed because of the nginx problem, despite mux didn't capture the param properly with the `params["chartName"]` it still was retrievable from the request itself. Now it is no longer necessary.